sdk/schemaregistry/schema-registry/review/schema-registry.api.md

```diff
@@ -0,0 +1,96 @@
+## API Report File for "@azure/schema-registry"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import * as coreHttp from '@azure/core-http';
+
+// @public (undocumented)
+export interface SchemaId {
+    // (undocumented)
+    id: string;
+}
+
+// @public (undocumented)
+export class SchemaRegistryClient extends SchemaRegistryClientContext {
+    constructor($host: string, options?: SchemaRegistryClientOptionalParams);
+    getIdBySchemaContent(groupName: string, schemaName: string, serializationType: string, schemaContent: string, options?: coreHttp.OperationOptions): Promise<SchemaRegistryClientGetIdBySchemaContentResponse>;
+    getSchemaById(schemaId: string, options?: coreHttp.OperationOptions): Promise<SchemaRegistryClientGetSchemaByIdResponse>;
+    registerSchema(groupName: string, schemaName: string, serializationType: string, schemaContent: string, options?: coreHttp.OperationOptions): Promise<SchemaRegistryClientRegisterSchemaResponse>;
+}
+
+// @public (undocumented)
+export class SchemaRegistryClientContext extends coreHttp.ServiceClient {
+    // (undocumented)
+    $host: string;
+    constructor($host: string, options?: SchemaRegistryClientOptionalParams);
+}
+
+// @public
+export interface SchemaRegistryClientGetIdBySchemaContentHeaders {
+    // (undocumented)
+    location?: string;
+    xSchemaId?: string;
+    xSchemaIdLocation?: string;
+    xSchemaVersion?: number;
+    xSerialization?: string;
+}
+
+// @public
+export type SchemaRegistryClientGetIdBySchemaContentResponse = SchemaRegistryClientGetIdBySchemaContentHeaders & SchemaId & {
+    _response: coreHttp.HttpResponse & {
```

Comment:
is it useful to return the `_response` property? should we elide it?

Reply:
(Artifact of generation, stopped exporting generated code)